tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -55,6 +55,14 @@ FilterPipeline::FilterPipeline()
     : max_chunk_size_(constants::max_tile_chunk_size) {
 }
 
+FilterPipeline::FilterPipeline(
```

Comment:
There's a new constructor pending merge that you'll need to use here instead of this one.
https://github.com/TileDB-Inc/TileDB/pull/2799/files#diff-5e358b038ca79fb45b4362d31c4f0d1961f4a11e15042f08e29546d3331448ec

There's a significant change in that PR as well. It changes from `tdb_new` to `shared_ptr`, so a lot of the detailed assumptions in code below will need to change.